grails-app/services/au/org/ala/merit/ProjectService.groovy

```diff
@@ -2052,4 +2059,8 @@ class ProjectService  {
     boolean canBulkRegenerateReports(Map project) {
         return project.reports.size() > 0
     }
+
+    boolean excludesNotApproved(Map report) {
```

Comment:
Is this used?  It also looks like a duplicate of the method in the ReportService